deploy/olm/23.4.0/manifests/stackable-spark-k8s-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,249 @@
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: spark-k8s-operator.v23.4.0
+spec:
+  annotations:
+    support: stackable.tech
+    olm.properties: '[]'
+    categories: Storage
+    capabilities: Full Lifecycle
+    description: Stackable Operator for Apache Spark
+    repository: https://github.com/stackabletech/spark-k8s-operator
+    containerImage: docker.stackable.tech/stackable/spark-k8s-operator:23.4.0
+
+  displayName: Stackable Operator for Apache Spark
+  description: |-
+    This is a Kubernetes operator to manage [Apache spark](https://spark.apache.org/) ensembles. The Stackable Apache Spark Operator
+    is part of the Stackable Data Platform, a curated selection of the best open source data apps like Kafka, Druid, Trino or Spark, all
+    working together seamlessly. Based on Kubernetes, it runs everywhere – on prem or in the cloud.
+
+    You can install the operator using [stackablectl or helm](https://docs.stackable.tech/spark-k8s/stable/getting_started/installation.html).
```

Comment:
It just occurred to me that this sentence doesn't make sense when reading it in the OS marketplace :)

Reply:
I'll try and remember to remove it en route to the certification PR :)